traffic_ops/traffic_ops_golang/server/servers.go

```diff
@@ -335,14 +335,12 @@ func getMidServers(servers []tc.ServerNullable, tx *sqlx.Tx) ([]tc.ServerNullabl
 	edgeIDs := strings.Join(ids, ",")
 	// TODO: include secondary parent?
 	q := selectQuery() + `
-WHERE s.id IN (
-	SELECT mid.id FROM server mid
-	JOIN cachegroup cg ON cg.id IN (
-		SELECT cg.parent_cachegroup_id
-		FROM server s
-		JOIN cachegroup cg ON cg.id = s.cachegroup
-		WHERE s.id IN (` + edgeIDs + `))
-	JOIN type t ON mid.type = (SELECT id FROM type WHERE name = 'MID'))
+JOIN type AS t ON s.type = t.id
```

Comment:
I'm getting the following error when trying to hit the endpoint that calls this query:

```
GET /api/2.0/servers?dsId=22 HTTP/1.1" 400
pq: table name "t" specified more than once
```